include/parser/expression-tokenizer.hpp

```diff
@@ -80,7 +80,7 @@ class ExpressionTokenizer {
     } else {
       // We are done, but there is an unrecognized token. We return as if the
       // string was empty.
-      state.addError("Unrecognized token at: " + temp.substr(0, 20),
+      state.addError(Translateable::createShared(P_TR("Unrecognized token at: %1"), {temp.substr(0, 20)}),
```

Comment:
I'd prefer putting the shared pointer on a separate line (with its own variable) rather than having everything inline